src/build_log.cc

```diff
@@ -293,7 +293,7 @@ LoadStatus BuildLog::Load(const string& path, string* err) {
         unlink(path.c_str());
         // Don't report this as a failure.  An empty build log will cause
```

Comment:
This comments needs to be updated / removed now, right?

Reply:
I think the comment is still correct when comparing it with the other `return LOAD_NOT_FOUND`. Maybe change "empty" into "missing"?